extension/public/popup.css

```diff
@@ -337,3 +337,20 @@ input:checked + .slider:before {
 .slider.round:before {
   border-radius: 50%;
 }
+
+/* Tooltips */
+.tooltip { 
+  visibility: hidden; 
+  position: absolute;
+  z-index: 1;
+  
+  background-color: #000;
+  color: #fff;
+  padding: 4px;
+  border-radius: 6px;
+  font-size: 14px;
```

Comment:
Looks great! I've noticed other websites usually have a different cursor when an element is interactable. [Examples](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor)

The default HTML cursor for tooltips is a bit unintuitive imo, (try it by pasting `data:text/html,<abbr title="Tooltip">Some text</abbr>` into the URL bar). I think `cursor: help` would work well here (?)

```suggestion
  font-size: 14px;
  cursor: help;
```